common/Storyboarding/OsbSpriteWriter.cs

```diff
@@ -0,0 +1,306 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using StorybrewCommon.Storyboarding.Commands;
+using StorybrewCommon.Storyboarding.CommandValues;
+using StorybrewCommon.Storyboarding.Display;
+
+namespace StorybrewCommon.Storyboarding
+{
+    class OsbSpriteWriter
+    {
+        protected OsbSprite OsbSprite;

```

Comment:
It's more of a general thing, but I'll write it here: private anything that can be private, lowercase anything that is private. readonly anything protected that the inheriting class isn't supposed to change.

Reply:
unless I overlooked something this should be fixed